docs/markdown/Native-environments.md

```diff
@@ -25,6 +25,30 @@ persistent environment:
 All of the rules about cross files and changed settings apply to native files
 as well, see [here](Cross-compilation.md#changing-cross-file-settings)
 
+## Site local configuration
+
+*New in 0.54.0*
+
+Meson allows system administrators, OS distributions, and users to set
+default policy using a special native file, `siteconfig.ini`. This is looked
+up in all of the search paths meson searches. This applies to Unix-like OSes
+only.
+
+It is recommended that a minimal number of settings be put in this config. A
+good thing to put in this are non standard paths, for example, if a
+distribution uses Debian style libdirs (`/usr/lib/<host-triple>`) they could
+use this config for x86_64 Linux with glibc:
+
+```ini
+[paths]
+libdir = 'x64_64-linux-gnu'
```

Comment:
Shouldn't this be `lib/x64_64-linux-gnu`?

Reply:
err, yeah it should